pkg/database/hdb/hdb.go

```diff
@@ -0,0 +1,28 @@
+package hdb
+
+import (
+	"context"
+	"database/sql"
+	"time"
+
+	_ "github.com/SAP/go-hdb/driver"
+)
+
+const (
+	MaxIdleConns    = 10
+	MaxOpenConns    = 10
+	MaxConnLifetime = 5 * time.Minute
```

Comment:
im sligtly confused by these constants being here.  i understand other packages might do this, but asking as we had more drivers.... seems like a wrong pattern hash-tag-bridge-too-far